### ISSUE TITLE
Color Scheme not saving in customizer.

### ISSUE THREAD

**Issue by kprovance** [reduxframework/redux-framework#4007]
It has to do with the unique way the extension saves data, which is not the traditional setting to save data, as there are many aspects of the data pieces.

**Comment by brimxbrand** [comment]
Color schema is still not working.

**Comment by kprovance** [comment]
No, in the customizer, it's not. I wouldn't get my hopes up about that ever being resolved.
